seaborn/categorical.py

```diff
@@ -1577,6 +1577,10 @@ def boxplot(
     fliersize=None, hue_norm=None, native_scale=False, log_scale=None, formatter=None,
     legend="auto", ax=None, **kwargs
 ):
+    
+    if "positions" in kwargs:
+        msg = "boxplot() does not support parameter 'positions'. Consider to use native_scale=True and specify positions via parameter 'x'."
```

Comment:
```suggestion
        msg = "boxplot does not support parameter `positions`. Consider using `native_scale=True`.
```
Nit: For a horizontal boxplot you'd specify the positions with `y` so let's make the suggestion less specific. Note that I've also tweaked the formatting.